server/src/controllers/timeline.js

```diff
@@ -1,15 +1,27 @@
-export const getAllTimeline = (req, res) => {
-    res.status(200).json({ 'status': 'fetch all timelines' });
-}
-export const getTimelineById = (req, res) => {
-    res.status(200).json({ 'status': 'fetch timeline id' });
-}
-export const postTimeline = (req, res) => {
-    res.status(200).json({ 'data': 'post timeline obj'});
-}
-export const updateTimeline = (req, res) => {
-    res.status(200).json({ 'data': 'update timeline obj'});
-}
-export const deleteTimeline = (req, res) => {
-    res.status(200).json({ 'status': 'deleted timeline_id'});
-}
\ No newline at end of file
+import * as timelineDao from '../daos/timeline';
+
+export const getAllTimeline = async (req, res) => {
+  const response = await timelineDao.getAllPublicTimeline();
+  res.status(200).json(response);
+};
+export const getTimelineById = async (req, res) => {
+  const { id } = req.params;
```

Comment:
why no error handling on any controllers ?

Reply:
error handling moved to data access files. since there i am doing CRUD on mongodb.